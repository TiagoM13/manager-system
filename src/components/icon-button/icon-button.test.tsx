import React from 'react';

import { Plus } from '@phosphor-icons/react';

import { render } from '@testing-library/react';

import { IconButton } from '.';

describe('<IconButton />', () => {
  test('should render correctly component', () => {
    const { getByTestId } = render(
      <IconButton>
        <Plus data-testid="icon" />
      </IconButton>,
    );

    expect(getByTestId('icon')).toBeTruthy();
  });
});
