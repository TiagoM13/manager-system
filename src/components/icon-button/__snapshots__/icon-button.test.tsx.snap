// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconButton /> should render correctly component 1`] = `
<DocumentFragment>
  <button
    class="undefined bg-white hover:bg-slate-200/50 border border-slate-400/50 rounded-md p-1.5 duration-300 disabled:bg-slate-300/50 disabled:hover:bg-slate-300/50 outline-slate-400"
  >
    <svg
      data-testid="icon"
      fill="currentColor"
      height="1em"
      viewBox="0 0 256 256"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M224,128a8,8,0,0,1-8,8H136v80a8,8,0,0,1-16,0V136H40a8,8,0,0,1,0-16h80V40a8,8,0,0,1,16,0v80h80A8,8,0,0,1,224,128Z"
      />
    </svg>
  </button>
</DocumentFragment>
`;
